R/metadata.R

```diff
@@ -276,6 +278,9 @@ static_orderly_dependency <- function(args) {
   query <- args$query
   files <- args$files
 
+  static_name <- static_string(name)
+  has_name <- !is.null(static_name) || is.null(name)
```

Comment:
Should there be `!` before `is.null(name)` ?

Reply:
no, here it means that "the name we have is final"